From 69a5047e931cb057dfe4e0e28557c255d0e0d47a Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 6 Nov 2018 14:57:23 +0100
Subject: [PATCH] [SQUSH][DNM] Merge tag 'android-9.0.0_r16' into lineage-16.0

commit a8e07aecb24898d7d2b49cb785b0c193a4b134b4
Author: Roger Liao <rogerliao@google.com>
Date:   Wed Jul 4 12:27:34 2018 +0800

    Android.mk: add sdm710

    Fixes the following build error:
    error: update_engine_sideload (EXECUTABLES android-arm64) missing
    bootctrl.sdm710 (STATIC_LIBRARIES android-arm64)

    Bug: 111163725
    Change-Id: I82f2321d580cb2fdb15d2343e39abed5ccda50b1

Change-Id: I6a5dd86adae4886c4847f1ab7bbf7716c1cfcf76
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 49efdae..51648f4 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,7 +1,7 @@
 ifeq ($(AB_OTA_UPDATER),true)
 # TODO:  Find a better way to separate build configs for ADP vs non-ADP devices
 ifneq ($(BOARD_IS_AUTOMOTIVE),true)
-ifneq ($(filter msm8953 msm8996 msm8998 sdm660 sdm845,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8953 msm8996 msm8998 sdm660 sdm845 sdm710,$(TARGET_BOARD_PLATFORM)),)
 LOCAL_PATH := $(call my-dir)
 
 # HAL Shared library for the target. Used by libhardware.
-- 
2.17.1

